www/src/data/sidebars/doc-links.yaml

```diff
@@ -188,26 +188,26 @@
         - title: Styling Your Site
           link: /docs/styling/
           items:
-            - title: Using Layout Components
-              link: /docs/layout-components/
-            - title: Using CSS Modules
+            - title: Standard styling with global CSS files
+              link: /docs/global-css/
+            - title: Locally-scoped styles with CSS Modules
               link: /docs/css-modules/
-            - title: Using Typography.js
-              link: /docs/typography-js/
-            - title: Using CSS-in-JS Library Emotion
-              link: /docs/emotion/
-            - title: Using CSS-in-JS Library Glamor
-              link: /docs/glamor/
-            - title: Using CSS-in-JS Library Styled Components
-              link: /docs/styled-components/
-            - title: Creating Global Styles
-              link: /docs/creating-global-styles/
-            - title: Writing Component CSS
```

Comment:
I think we should keep the component CSS page intact, as it's an important workflow.

Reply:
Resolving, [discussing above](https://github.com/gatsbyjs/gatsby/pull/13766#discussion_r281635189)